src/task/SetRemoteDescriptionTask.ts

```diff
@@ -46,9 +47,8 @@ export default class SetRemoteDescriptionTask extends BaseTask {
       this.context.videoSendCodecPreferences.length > 0
     ) {
       sdp = new SDP(sdp).withVideoSendCodecPreferences(
-        this.context.meetingSupportedVideoSendCodecPreferences !== undefined
-          ? this.context.meetingSupportedVideoSendCodecPreferences
-          : this.context.videoSendCodecPreferences
+        this.context.meetingSupportedVideoSendCodecPreferences ??
```

Comment:
I checked the support for `??` operator and it seems mostly its fine but some Chrome and Edge version will fail to parse it:
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing
- https://docs.aws.amazon.com/chime-sdk/latest/dg/meetings-sdk.html#mtg-browsers

Until we bump that documentation probably we should still fallback to older version :)